### PR TITLE
New State Tree Layout

### PR DESCRIPTION
**Pull request by s0undt3ch**

### CODE REVIEW

File: os-images/AWS/debian/scripts/apt.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Acquire::http::Pipeline-Depth 0;
+Acquire::http::No-Cache true;
+Acquire::BrokenProxy    true;
+EOF
+
+# Uninstall unattended-upgrades
+apt-get purge -y unattended-upgrades
+
+# Need to limit arch for default apt repos due to
+# https://github.com/actions/runner-images/issues/1961
```
Comment (Ch3LL) [inline os-images/AWS/debian/scripts/apt.sh:34]: Not required for merge, but do we need all of these extra commands in comments?

Comment (s0undt3ch) [reply os-images/AWS/debian/scripts/apt.sh:34]: I wanted to keep the file as close as to what it was when I copied it. That's the only reason for those comments.

File: os-images/AWS/ubuntu/scripts/apt.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Acquire::http::Pipeline-Depth 0;
+Acquire::http::No-Cache true;
+Acquire::BrokenProxy    true;
+EOF
+
+# Uninstall unattended-upgrades
+apt-get purge -y unattended-upgrades
+
+# Need to limit arch for default apt repos due to
+# https://github.com/actions/runner-images/issues/1961
```
Comment (Ch3LL) [inline os-images/AWS/ubuntu/scripts/apt.sh:34]: same case here

Comment (s0undt3ch) [reply os-images/AWS/ubuntu/scripts/apt.sh:34]: Same reason as above.

**Comment by s0undt3ch** [comment]
I'm actually thinking on pushing these changes as the `main` branch, which will then become the default branch....

File: os-images/files/provision-system.sh [hunk 3]
```diff
@@ -8,6 +8,18 @@ set -x
 echo "Contents of ${SALT_ROOT_DIR}:"
 ls -lah ${SALT_ROOT_DIR}
 
-COMMAND="salt-call --config-dir=${SALT_ROOT_DIR}/conf --local --log-level=info --file-root=${SALT_ROOT_DIR}/states --pillar-root=${SALT_ROOT_DIR}/pillar state.sls ${SALT_STATE} --retcode-passthrough"
+if [ -f /tmp/salt/bin/salt-call ]; then
+    SALT_CALL=/tmp/salt/bin/salt-call
+else
+    echo "Could not find a Salt mayflower build or Salt install in a PyEnv environment"
```
Comment (Ch3LL) [inline os-images/files/provision-system.sh:14]:
```suggestion
    echo "Could not find a Salt relenv build or Salt install in a PyEnv environment"
```

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Review by Ch3LL — DISMISSED** [review]

**Pull request merged**